src/appPackageManifest.ts

```diff
@@ -68,31 +75,62 @@ export default function appPackageManifest({ projectConfig, buildId } : {
           return next(new PluginError(PLUGIN_NAME, ex, { fileName: file.relative }));
         }
 
+        function throwDuplicateComponent(existingPath: string) {
+          next(
+            new PluginError(
+              PLUGIN_NAME,
+              `Duplicate ${bundleInfo.type} component bundles: ${file.relative} / ${existingPath}`,
```

Comment:
This error message would be more informative if it included the family name for a colliding device bundle. E.g.:

    Colliding device/higgs component bundles: foo.zip / bar.zip